content-repo/extra-docs/releases/20.11.0.md

```diff
@@ -0,0 +1,1000 @@
+# Cortex XSOAR Content Release Notes for version 20.11.0 (178169)
+##### Published on 10 November 2020
+
+### Breaking Changes
+Several packs include breaking changes.
+
+- [Microsoft Cloud App Security Pack v1.0.9](#Microsoft-Cloud-App-Security-Pack-v1.0.9)
```

Comment:
Links are wrong, should be for example `#microsoft-cloud-app-security-pack-v109`